README.md

```diff
@@ -20,14 +20,16 @@ The Docker compose file at `test/docker-compose.yml` can be used to run the requ
 
 | name | part of | description | required | default value |
 |------|---------|-------------|----------|---------------|
-|`client_id`|source|The Client ID for Marketo Instance|true| |
-|`client_secret`|source|The Client Secret for Marketo Instance|true| |
-|`endpoint`|source|The Endpoint for Marketo Instance|true||
+|`ClientID`|source|The Client ID for Marketo Instance|true| |
+|`ClientSecret`|source|The Client Secret for Marketo Instance|true| |
+|`ClientEndpoint`|source|The Endpoint for Marketo Instance|true||
```

Comment:
We should have it camel case, starting with a lower case letter.

Reply:
updated.